pages/view/contact/_id.vue

```diff
@@ -99,17 +86,8 @@ export default {
     AppButton
   },
 
-  async fetch() {
-    try {
-      await this.$store.dispatch('contacts/fetchContact', this.$route.params.id)
-    } catch (e) {
-      // eslint-disable-next-line no-console
-      console.log('Error: ', e.response)
-    }
-  },
-
   computed: mapState({
-    contactInfo: state => state.contacts.contact
+    contactInfo: state => state.contacts.contacts.find((contact) => { return contact._id === window.$nuxt.$route.params.id })
```

Comment:
i was wondering how to get the id  
   Nuxt

Reply:
Yeah that was a big hassle, figuring out how to get access to the $route object. Seems like it doesn't have access to "this" from within mapState/computed property!

Reply:
smart fellers :p